sda_download/sda_download.go

```diff
@@ -0,0 +1,259 @@
+package sdadownload
+
+import (
+	"encoding/json"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/NBISweden/sda-cli/helpers"
+	"github.com/vbauerster/mpb/v8"
+	"github.com/vbauerster/mpb/v8/decor"
+)
+
+// Help text and command line flags.
+
+// Usage text that will be displayed as command line help text when using the
+// `help download` command
+var Usage = `
+USAGE: %s sda-download -config <s3config-file> -dataset <datasetID> -url <uri> (-outdir <dir>) [filename(s)]
```

Comment:
We should make the cli read the token from ENV since the config file is mainly for uploading data and that can hav ea resigned token.

Reply:
Done

Reply:
Yes we can do that but on the other hand I think that is easier for the common user to to just download a file and then not bother about anything else. We can discuss it tomorrow with the rest of the team.